web/src/app.dev.js

```diff
@@ -12,38 +12,46 @@ import thunk from 'redux-thunk';
 import { initI18n } from './i18n';
 import reducer from './reducers';
 import Root from './components/Root';
-
-// Set locale based on browser language
-// if it matches one of our SUPPORTED_LOCALES
-// otherwise, set to DEFAULT_LOCALE ("en")
-initI18n();
-
-const history = createHistory();
-
-const middleware = [thunk, routerMiddleware(history), createLogger()];
-
-/* eslint-disable no-underscore-dangle */
-const composeEnhancers = window.__REDUX_DEVTOOLS_EXTENSION_COMPOSE__ || compose;
-/* eslint-enable */
-const store = createStore(
-  reducer(history), 
-  composeEnhancers(
-    applyMiddleware(...middleware),
-  )
-);
-
-const render = (Component, props) => {
-  ReactDOM.render(
-    <AppContainer>
-      <Component {...props} />
-    </AppContainer>,
-    document.getElementById('app')
+import { browserIsRed } from './util/browser';
+import { html } from './components/UpgradeBrowser';
+
+if (browserIsRed) {
+  document.getElementById('app').innerHTML = `
```

Comment:
oh boy, does that give me flashbacks to the bad old days!

Reply:
Yeah... It's not exciting to revert but who knows what those dinosaur browsers will do otherwise 😬